bindings/python/tests/test_iterators.py

```diff
@@ -0,0 +1,40 @@
+# type: ignore
+from slimt import iterators
+
+def test_iterators(service, models):
+    source = "Hi, How are you? Its been a long time.\nCan you help me out with some things?"
+    model = models[1]
+    response = service.translate(model, [source], html=False)[0]
+
+    target = response.target
+    text = target.text
+
+    sen_iter_tgt = iterators.sentences(target)
+    word_iter_global = iterators.words(target)
```

Comment:
We do not prefer abbreviations when it can be named better.

```py
sentences = iterators.sentences(response.target)
for sentence in sentences:
   # ...
   
words = iterators.words(response.target)
for word in words:
   # ...
```

 In general chaining more entities into the naming connected by `_` points to that we've failed somewhere at abstracting (Imagine `sentence`, `sentence_item` - instead of `sentence`, `word` for an example).